src/components/pages/tags/FormPage.jsx

```diff
@@ -1,25 +1,40 @@
 import { Button, Form } from 'react-bootstrap';
-import { useNavigate } from 'react-router-dom'
+import { useParams, useNavigate } from 'react-router-dom'
 import TagService from '@services/tagService';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 
 function FormPage() {
     const [name, setName] = useState('');
```

Comment:
It works, but with this approach we need to create a useState for every field in the form, change this to use only one useState on a object, EG:
const [formData, setFormData] = useState({
  name: ''
});

Reply:
We'll also need to update the useEffect